.vscode/settings.json

```diff
@@ -0,0 +1,4 @@
+{
+    "editor.tabSize":2,
+    "files.insertFinalNewline": true
+}   
```

Comment:
Me thinks you may want to add ".vscode" to the .gitignore, or possibly your global .gitignore file. (https://help.github.com/articles/ignoring-files/)